### PR TITLE
AC Automated Test - LRAC-8319 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-8319
https://issues.liferay.com/browse/LRAC-8744 (refactor ticket)

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase [hunk 1]
```diff
@@ -30,12 +30,60 @@ definition {
...
-	@ignore = "true"
 	@priority = "5"
 	test CustomRangeTimePeriodFilterSites {
+		property portal.upstream = "quarantine";
+
+		// AC Refactor: the test needs data from a past date
+		// AC Refactor ticket: LRAC-8744
+
+		task ("Site and page creation and connection with AC") {
+			ACUtils.addSiteAndPage(layoutName = "AC Page");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:41]: Hi @rafaellajordao, it is not necessary to pass parameter to use the page as **AC Page**

Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesPropertyAnalyticsFilter.testcase:41]: Yeah, this parameter is not strictly needed, but we can leave as is now and change it when refactoring.

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**